tox.ini

```diff
@@ -25,72 +25,68 @@ passenv =
 commands =
     py.test --cov=scrapy --cov-report= {posargs:scrapy tests}
 
-[testenv:trusty]
+[testenv:py27-pinned-versions]
 basepython = python2.7
 deps =
-    pyOpenSSL==0.13
-    lxml==3.3.3
-    Twisted==13.2.0
-    boto==2.20.1
-    Pillow==2.3.0
-    cssselect==0.9.1
-    zope.interface==4.0.5
-    -rtests/requirements-py2.txt
-
-[testenv:jessie]
-# https://packages.debian.org/en/jessie/python/
-# https://packages.debian.org/en/jessie/zope/
-basepython = python2.7
-deps =
-    cryptography==0.6.1
-    pyOpenSSL==0.14
-    lxml==3.4.0
-    Twisted==14.0.2
-    boto==2.34.0
-    Pillow==2.6.1
+    -ctests/constraints.txt
     cssselect==0.9.1
-    zope.interface==4.1.1
```

Comment:
I see some previously-pinned libraries are no longer pinned, namely `cryptography`, `Pillow` and `zope.interface`. Maybe we should pin them as well?

As for boto/botocore, if we are dropping support for boto maybe we should update the documentation accordingly (see https://github.com/scrapy/scrapy/pull/3294/files).

Reply:
True, those dependencies were only pinned in the Debian env, so they were deleted along with it. I can add them to the requirements files and the pinned deps lists. Thanks for noticing.